assembl/templates/base.jinja2

```diff
@@ -6,20 +6,46 @@
     <title>{% block page_title %}Assembl{% endblock %}</title>
     <meta name="viewport" content="width=device-width, initial-scale = 1,user-scalable=no,maximum-scale=1.0">
 
-    {% block meta_tags %}
-    <meta name="description" content="">
-    <meta property="og:title" content="Assembl" />
-    <meta property="og:type" content="website" />
-    <meta property="og:url" content="{{ application_url }}" />
-    <meta property="og:image" content="{{ application_url }}/static/css/themes/default/img/crowd2.jpg" />
-    <meta property="og:image:width" content="1600" />
-    <meta property="og:image:height" content="222" />
-    <meta property="og:image" content="{{ application_url }}/static/css/themes/default/img/logo.png" />
-    <meta property="og:image:width" content="111" />
-    <meta property="og:image:height" content="35" />
-    <meta name="robots" content="noindex" />
-    <meta name="googlebot" content="noindex" />
-    {% endblock meta_tags %}
+    {% block meta_tags_public %}
+        {% if not private_social_sharing %}
+            <meta name="description" content="{{get_description}}">
+            <meta property="og:title" content="{{get_topic}}" />
+            <meta property="og:type" content="website" />
+            <meta property="og:url" content="{{ application_url }}" />
+            <meta property="og:image" content="{{get_landing_page_image}}" />
+            <meta property="og:image:width" content="1600" />
+            <meta property="og:image:height" content="222" />
+            <meta property="og:locale" content="{{opengraph_locale}}" />
+            <meta name="robots" content="noindex" />
+            <meta name="googlebot" content="noindex" />
+
+            <meta name="twitter:card" content="summary_large_image">
+            <meta name="twitter:site" content="@bluenove">
```

Comment:
Are you sure we can't provide a more custom per-debate value for twitter:site and twitter:creator?

Reply:
well theoritically I can add whatever value I want, or write a function that renders any result that I want ... but I don't have an idea on what should be put instead ... what do you propose ?

Reply:
Well, `@bluenove` isn't valid here either. It should be the url of the discussion. This can be fetched from the discussion object as `{{discussion.get_discussion_urls()[1]}}` which would deliver the `https` based url of the discussion.